setup.cfg

```
@@ -23,6 +23,7 @@ max-complexity = 10
 max-line-length = 100
 show_source = True
 statistics = True
+extend-ignore = I100, I101, I201, I202
```

For the sake of clarity/self-documentation, how about something like this?

```suggestion
extend-ignore = I100, I101, I201, I202  # ignore errors thrown by flake8-import-order (if it is installed), which is superceded by isort
```